composer.json

```diff
@@ -24,23 +24,23 @@
             "Tests/Fixtures/app/AppKernel.php",
             "Tests/Fixtures/ErrorHandler.php"
             ],
-        "psr-4": { "Tests\\": "tests/"}
+        "psr-4": { "Tests\\": "Tests/"}
     },
     "require": {
         "php": ">=7.1",
-        "rollbar/rollbar": "^2",
-        "symfony/dependency-injection": "^5.0",
-        "symfony/config": "^5.0",
-        "symfony/http-kernel": "^5.0",
-        "symfony/monolog-bundle": "*",
-        "symfony/serializer": "*",
+        "rollbar/rollbar": "^2|^3.0",
```

Comment:
rollbar/rollbar ^3 is only allow with `"php": ">=8.0.0 <9.0",` won't it make a conflict?

Reply:
> rollbar/rollbar ^3 is only allow with `"php": ">=8.0.0 <9.0",` won't it make a conflict?

No. Because it can use rollbar/rollbar ^2